slack-infra/resources/ingress.yaml

```diff
@@ -25,3 +25,8 @@ spec:
             backend:
               serviceName: slack-welcomer
               servicePort: 80
+          - path: /infra/post-message/*
+            backend:
+              serviceName: slack-post-message
+              servicePort: 80
+              
```

Comment:
now it's one too much I think :-)

Reply:
We need only one new line right?

Reply:
Sorry! I had some new line settings issue with my editor. I have edited correctly now.

Reply:
Looks good now. Thank you! :-)